src/components/ui/date-range-picker/date-range-picker.tsx

```diff
@@ -5,23 +5,25 @@ interface DateRangePickerProps {
   label?: string
   value: DateValueType
   onChange: (value: DateValueType, e?: HTMLInputElement | null | undefined) => void
+  error?: { start_date?: string | null; end_date?: string | null }
 }
 
 export const DateRangePicker: React.FC<DateRangePickerProps> = ({
   name,
   label,
   value,
   onChange,
+  error = {},
 }: DateRangePickerProps) => {
   const currentDate = new Date()
   const minDate = new Date(currentDate)
   minDate.setFullYear(currentDate.getFullYear() - 50)
   const maxDate = new Date(currentDate)
   maxDate.setFullYear(currentDate.getFullYear() + 50)
-
   const containerClassName = (defaultClassName: string) => {
-    const customStyle = "border rounded-md"
-    return `${defaultClassName} ${customStyle}`
+    const customStyle =
+      error.start_date !== undefined && error.end_date !== undefined ? "border-red-500" : ""
```

Comment:
I think this should be `OR` instead of `AND`?

Reply:
Okeyy changed :3